torch/testing/_internal/common_methods_invocations.py

```diff
@@ -91,6 +95,16 @@ def __init__(self,
 
         self.skips = skips
         self.decorators = decorators
+        self.output_func = output_func
+
+        self.is_autodiffed = is_autodiffed
```

Comment:
Add an addmm OpInfo below with one sample input to test the fusible nodes case

Reply:
We read the addmm method test wrong, it has two nonfusible nodes instead of fusible nodes. So adding this op wouldn't help test out the fusible node case ☹️

Reply:
Doh!

Want to pick a different op?

Reply:
Also - do we need an example with two nonfusible nodes to test with, too? I forget the details.

Reply:
So, I wasn't able to find any actual test that uses the 'fusible node case' in test_jit.py or elsewhere in the repository... I'm not sure why it was a part of the original assertAutodiffNode() function. I can take it out? @eellison Do you know if keeping this unused code would be helpful?

Reply:
I can add the addmm test for the sake of testing the two nonfusible nodes in OpInfo if you think it's necessary. I'm not sure really why it would be. Addmm is the only op in the original method tests that has more than one nonfusible node. Other tests in test_jit.py have more than one nonfusible node but nothing in OpInfo

Reply:
It'd be nice to see all the new fields, including `autodiff_nonfusible_nodes` and `autodiff_fusible_nodes`, actually get used. It doesn't have to be part of this PR if that would cause a delay.

That's wild that are no ops testing for fusions. If that's the case this PR could mark the relevant code with a comment like, "NOT CURRENTLY USED" or "Intended to be used when..."

Reply:
I added notes in the assertAutoDiff code and on the OpInfo class mentioning how the fusible_node case isn't used currently. I also added addmm to the commit above this one, I had to do some refactoring on how the sample inputs were being unpacked so it turned into a larger project